app/lib/widgets/app_bar/app_bar.dart

```diff
@@ -0,0 +1,126 @@
+import 'package:flex_ui/flex_ui.dart';
+import 'package:flex_ui/tokens/sizes.dart';
+import 'package:flutter/material.dart';
+import 'package:widgetbook/widgetbook.dart';
+import 'package:widgetbook_annotation/widgetbook_annotation.dart' as widgetbook;
+
+class FlexAppBar extends StatelessWidget implements PreferredSizeWidget {
```

Comment:
Reminder for us to discuss documentation of widgets

Reply:
Agree! I'll add one